frontend/src/scenes/experiments/Experiment.tsx

```diff
@@ -1,40 +1,48 @@
 import SaveOutlined from '@ant-design/icons/lib/icons/SaveOutlined'
 import { Button, Carousel, Col, Form, Input, Row } from 'antd'
-import { useActions, useValues } from 'kea'
+import { BindLogic, useActions, useValues } from 'kea'
 import { PageHeader } from 'lib/components/PageHeader'
 import { PropertyFilters } from 'lib/components/PropertyFilters'
 import { TaxonomicFilterGroupType } from 'lib/components/TaxonomicFilter/types'
 import React, { useRef } from 'react'
+import { InsightContainer } from 'scenes/insights/InsightContainer'
+import { insightLogic } from 'scenes/insights/insightLogic'
+import { SceneExport } from 'scenes/sceneTypes'
 import { userLogic } from 'scenes/userLogic'
 import { PropertyFilter } from '~/types'
 import './Experiment.scss'
 import { experimentLogic } from './experimentLogic'
 
+export const scene: SceneExport = {
```

Comment:
aren't most of the changes in this PR from https://github.com/PostHog/posthog/pull/7584? It's hard to see which code changes are from this PR vs the old one here 😞

Reply:
Hmm, whoops, it was based on top of the other one. Give me a moment, let me rebase!